### PR TITLE
RavenDB-16419 Fix error handling of IdleFlushTimer 

### PR DESCRIPTION
**Pull request by arekpalinski**

### CODE REVIEW

**Review by ppekrol — APPROVED** [review]

File: src/Raven.Server/Documents/Handlers/Debugging/StorageHandler.cs [hunk 1]
```diff
@@ -18,6 +18,52 @@ namespace Raven.Server.Documents.Handlers.Debugging
...
+            var env = Database.GetAllStoragesEnvironment()
+                .FirstOrDefault(x => string.Equals(x.Name, name, StringComparison.OrdinalIgnoreCase) && x.Type == type);
+
+            if (env == null)
+            {
+                HttpContext.Response.StatusCode = (int)HttpStatusCode.NotFound;
+                return Task.CompletedTask;
+            }
+
+            env.Environment.SuggestSyncDataFile();
```
Comment (ayende) [inline src/Raven.Server/Documents/Handlers/Debugging/StorageHandler.cs]: Should be force here, not suggest

File: src/Voron/StorageEnvironment.cs [hunk 2]
```diff
@@ -158,9 +161,6 @@ public StorageEnvironment(StorageEnvironmentOptions options)
                 else // existing db, let us load it
                     LoadExistingDatabase();
 
-                if (_options.ManualFlushing == false)
-                    Task.Run(IdleFlushTimer);
-
```
Comment (ayende) [inline src/Voron/StorageEnvironment.cs]: Why did you remove this?

File: src/Raven.Server/Documents/Handlers/Debugging/StorageHandler.cs (hunk 1 above)
Comment (arekpalinski) [reply src/Raven.Server/Documents/Handlers/Debugging/StorageHandler.cs]: Fixed in https://github.com/ravendb/ravendb/pull/11932/commits/bdf631647072d6fc969bf85e3ab27f0a6d352ac6

File: src/Voron/StorageEnvironment.cs (hunk 2 above)
Comment (arekpalinski) [reply src/Voron/StorageEnvironment.cs]: Since we moved the creation of the task to `NewLowLevelTransaction()` then at this point it's already created. We create a new transaction when opening or creating the storage.

Added `Debug.Assert` in https://github.com/ravendb/ravendb/pull/11932/commits/6d7e32b3c1f836bc0871e7ba5e1bee94248057d8
